u8timeseries/timeseries.py

```diff
@@ -504,7 +511,8 @@ def from_dataframe(df: pd.DataFrame,
     def from_times_and_values(times: pd.DatetimeIndex,
                               values: np.ndarray,
                               confidence_lo: np.ndarray = None,
-                              confidence_hi: np.ndarray = None) -> 'TimeSeries':
+                              confidence_hi: np.ndarray = None,
+                              freq: Optional[str] = None) -> 'TimeSeries':
```

Comment:
How about the `fill_missing_dates` argument?

Reply:
done